app/templates/server/config/express.js

```diff
@@ -13,8 +13,9 @@ var methodOverride = require('method-override');
 var cookieParser = require('cookie-parser');
 var errorHandler = require('errorhandler');
 var path = require('path');
+var lusca = require('lusca');
 var config = require('./environment');<% if (filters.auth) { %>
-var passport = require('passport');<% } %><% if (filters.twitterAuth) { %>
+var passport = require('passport');<% } %>
 var session = require('express-session');<% if (filters.mongoose) { %>
 var mongoStore = require('connect-mongo')(session);
 var mongoose = require('mongoose');<% } %><% } %>
```

Comment:
If you're removing `<% if (filters.twitterAuth) { %>` then you also need to remove the closing tag `<% } %>` here.